web_src/less/_repository.less

```diff
@@ -860,7 +860,7 @@
           border-radius: 50%;
           display: flex;
           float: left;
-          margin-left: -34px;
+          margin-left: -33px;
```

Comment:
Slipped this in since noticed that items are not aligned on center perfectly.

Reply:
May need to use a odd icon width for perfect centering.